doc/api/plotting/conv_func.rst

```diff
@@ -12,3 +12,4 @@ routines in PyVista.
    plot_arrows
    set_plot_theme
    create_axes_orientation_box
+   create_axes_marker
```

Comment:
Now that this has been added to the documentation, it is found that the docstring for this function does not comply with numpydoc format.  See:

https://github.com/pyvista/pyvista/actions/runs/3198138277/jobs/5222284301#step:11:772